RELEASE_NOTES

```diff
@@ -0,0 +1,116 @@
+Release 0.9.0 (Draft)
+=====================
+
+Changes
+-------
+
+Changes that may break users' config files have been annotated with a `(!)`.
+
+- Note: mpv is not compatible with Lua 5.3. Lua 5.1 or 5.2 is required.
+- The minimum required libass version is now 0.12.1 or newer.
+- The minimum required FFmpeg version is now 2.4.0 (equiv. Libav 11) or newer.
+- `(!)` The internal libmpg123 support was removed.
+- `(!)` The LIRC support was removed. Configure LIRC remotes as input devices instead.
+- `(!)` The Linux Joystick support was removed.
+- `(!)` `--ass-use-margins` has been renamed to `--sub-use-margins` and applies
+  only to plain-text (non-ASS) subtitles (enabled by default). The new
+  `--ass-force-margins` option applies only to ASS subtitles (disabled by
+  default). To get the old behaviour back, enable both at the same time.
+- `(!)` The `--sub-scale-with-window` option now only applies to plain text
+  (non-ASS) subtitles (enabled by default). The new `--ass-scale-with-window`
+  option does the same but only with ASS subtitles (disabled by default).
+- `(!)` The range for the `param1` for the `gaussian` `vo_opengl` scaler has
+  been redefined. Instead of being an arbitrary 1-100 range, have a default
+  value of 1.0, and anything higher is blurrier.
+- `(!)` The `seek`, `playlist_next, `playlist_prev`, `loadfile` and `loadlist`
+  parameters no longer accept numerical parameters where symbolic parameter
+  names exist.
+- `(!)` `vo_opengl` changes:
+  - `(!)` The `smoothmotion` suboption has been renamed to `interpolation`.
+    The old name is still supported for now.
+  - `(!)` The `bilinear_slow` scaler has been renamed to `triangle`.
+  - `(!)` `scale-down` has been renamed to `dscale` and now has its own set of
+    config options (e.g. `dscale-radius`).
+  - `(!)` Scaler radius no longer defaults to `3` but to a preferred value
+    that may be different for each filter.
+  - The `scale-radius` option may now go down as low as `0.5`, which is the
+    value used by the `nearest` filter.
+  - `spline36` is the new `cscale` default for `opengl-hq`. This might break
+    setups that use `fbo-format=8` (or less). To work around it, leave
+    `fbo-format` as its default, or set to something higher than 8, or set
+    `cscale=bilinear`, the previous default.
```

Comment:
I don't think `fbo-format=8` was valid (not with that exact value). Also, I'm unsure how a different fbo-format could break it. At worst you'd lose some quality due to lack of precision?